src/pages/SplitPosition/Form.tsx

```diff
@@ -292,12 +293,18 @@ export const Form = (props: Props) => {
     setConditionId(row.id)
   }, [])
 
+  useEffect(() => {
+    setResetFilterUpdate(!resetFilterUpdate)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [status])
+
```

Comment:
This seems to toggle off/on at each change in online status. If the user enters the form as a connected user, this run at least once I think and then filters are set to reset, but this in development happens twice so it does what it says it does when is offline and change to online. So the first time is kind of idempotent but only in non-production environments (we should test it to be sure), but I think this could break in production envs. 

Maybe I'm missing something, so if it passes QA it' is OK for me. If not, some functionality related to filters (like this one, or only this one) can be moved to a hook that composes with the status hook and is used to receive arbitrary callbacks.

Reply:
I generated a build, and test it, and seems to be working fine